src/lib/config.ts

```diff
@@ -14,6 +14,8 @@ interface Options {
 
 type UserConfig = {
   includePrivate: boolean;
+  address: string;
```

Comment:
ここのプロパティの名前がそのまま`qiita.config.json`の設定名になるので、`host`のほうが伝わりやすいかなと思ったのですが、いかがでしょうか？

```suggestion
  host: string;
```